llama_stack/apis/providers/providers.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+from typing import List, Protocol, runtime_checkable
+
+from pydantic import BaseModel
+
+from llama_stack.distribution.datatypes import Provider
+from llama_stack.schema_utils import json_schema_type, webmethod
+
+
+@json_schema_type
+class ProviderInfo(BaseModel):
+    api: str
+    provider_id: str
+    provider_type: str
+
+
+class GetProviderResponse(BaseModel):
```

Comment:
There's is an error with OpenAPI spec where `GetProviderResponse` is not actually defined.